### PR TITLE
Improve SchemaLoader tests

### PR DESCRIPTION
**Pull request by markstory**
Use a separate SQLite file to test table drop/load operations without mocks.

### CODE REVIEW

File: tests/TestCase/TestSuite/Fixture/SchemaLoaderTest.php [hunk 1]
```diff
@@ -89,32 +90,29 @@ public function testLoadMissingFile(): void
...
+        $path = TMP . 'schema_loader_test.sqlite';
+        ConnectionManager::setConfig('test_schema_loader', [
+            'className' => Connection::class,
+            'driver' => Sqlite::class,
+            'database' => $path,
+        ]);
+        $loader = new SchemaLoader(['outputLevel' => ConsoleIo::QUIET]);
+
+        $schemaFile = $this->createSchemaFile('schema_loader_first');
+        $loader->loadFiles($schemaFile, 'test_schema_loader');
```
Comment (othercorey) [inline tests/TestCase/TestSuite/Fixture/SchemaLoaderTest.php:103]:
```suggestion
        $this->loader->loadFiles($schemaFile, 'test_schema_loader');
```

File: tests/TestCase/TestSuite/Fixture/SchemaLoaderTest.php [hunk 2]
```diff
@@ -89,32 +90,29 @@ public function testLoadMissingFile(): void
...
+        $connection = ConnectionManager::get('test_schema_loader');
+
+        $result = $connection->getSchemaCollection()->listTables();
+        $this->assertEquals(['schema_loader_first'], $result);
+
+        $schemaFile = $this->createSchemaFile('schema_loader_second');
+        $loader->loadFiles($schemaFile, 'test_schema_loader');
+
+        $result = $connection->getSchemaCollection()->listTables();
+        $this->assertEquals(['schema_loader_second'], $result);
```
Comment (othercorey) [inline tests/TestCase/TestSuite/Fixture/SchemaLoaderTest.php:113]: This needs to test the drop and truncate tables feature.

Comment (othercorey) [reply tests/TestCase/TestSuite/Fixture/SchemaLoaderTest.php:113]: Yea , it does. I missed that it was an equals check.

Comment (markstory) [reply tests/TestCase/TestSuite/Fixture/SchemaLoaderTest.php:112]: I added an insert to the generated SQL file and assertions for the truncation.

**Pull request merged**